app/javascript/components/App.js

```diff
@@ -76,18 +76,22 @@ class App extends React.Component {
             path="/fatenew"
             render={(props) => <FateNew createNewEvent={this.createNewEvent} />}
           />
-          <Route path="/fateshow/:id" render={(props) => { 
-            let id = props.match.params.id
-            let event = this.state.events.find(event => event.id === +id)
-            {console.log(this.state.events);}
-            return <FateShow event={event}/>
-          }} />
-          <Route path="/aboutus" component={AboutUs}/>
+          <Route
+            path="/fateshow/:id"
+            render={(props) => {
+              let id = props.match.params.id;
+              let event = this.state.events.find((event) => event.id === +id);
+              {
+                console.log(this.state.events);
```

Comment:
same here

Reply:
Removed it, while yes it still worked was useless after we were done with it